### PR TITLE
Improve text template with icon

### PR DESCRIPTION
**Pull request by neihousaigaai**

### CODE REVIEW

File: commands/command.py [hunk 1]
```diff
@@ -182,56 +31,40 @@ async def parse_command(client, game, message):
...
 
             if is_valid_channel:
                 author = message.author
-                is_valid_command = False
-
-                if cmd == "ship":
-                    if len(parameters) == 2:
-                        if len(message.raw_mentions) == 2:
-                            msg = await game.do_player_action(cmd, author.id, *message.raw_mentions)
+                required_param_number = 2 if cmd == "ship" else 1
```
Comment (nitro2) [inline commands/command.py:46]: What difference between this `required_param_number ` and the one in json/command_info.json?

**Comment by nitro2** [comment]
Any guide if I want to add new command? Which places should I consider?

File: commands/command.py (hunk 1 above)
Comment (neihousaigaai) [reply commands/command.py:46]: oops, I forgot to import it from json file :smile:

**Comment by neihousaigaai** [comment]
A command information looks like this:
```json
"cmd_name": {
  "description": {
    "vi": "description_in_vietnamese",
    "en": "description_in_english"
  },
  "required_param_number": number_of_required_parameters_to_the_command.
}
```
For example:
- `!join` --> required_param_number = 0 (**no** parameters)
- `!vote 1` --> required_param_number = 1 (**must** have to vote 1 player)
- `!ship 1 2` --> required_param_number = 2 (**must** have to ship 2 players)

**Review by nitro2 — APPROVED** [review]

**Pull request merged**